services/web/pkg/config/config.go

```diff
@@ -100,7 +100,7 @@ type ExternalApp struct {
 
 // ExternalAppConfig defines an external web app configuration.
 type ExternalAppConfig struct {
-	URL string `json:"url,omitempty" yaml:"url" env:"" introductionVersion:"pre5.0"`
+	URL string `json:"url,omitempty" yaml:"url" env:"" desc:"" introductionVersion:"pre5.0"`
```

Comment:
This had an `env` section that is empty. 
I added an empty `desc` section to keep the checker happy.
Should I delete them both?

Reply:
```suggestion
	URL string `json:"url,omitempty" yaml:"url"`
```

Reply:
Only configurable via yaml.